src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int BOUND = 100;
+    private static final int DEFAULT_RADIUS = 100;
+    private static final String DEFAULT_COLOR = Color.WHITE.name();
+    private final ColorSupplier colorSupplier = new ColorSupplier();
+    private final Random random = new Random();
+    private final int quantityOfFigures = 5;
+
+    public Figure getRandomFigure() {
+        return switch (random.nextInt(quantityOfFigures)) {
+            case (0) -> getCircle();
```

Comment:
```suggestion
            case (0) -> getCircle(color);
```

same for other figures